src/populate.py

```diff
@@ -0,0 +1,137 @@
+import os
+
+import django
+
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "vikoverflow.settings.local")
+django.setup()
+
+from faker import Faker
+import random
+from common.models import Votes
+from django.contrib.auth.models import User
+from account.models import Profile
+from django.core.management import call_command
+from question.models import Question, Answer, Comment
+import datetime
+
+call_command('flush')
+
+fake = Faker('hu_HU')
+users = []
+
+# Create admin user
+admin = User()
+admin.username = 'admin'
+admin.is_superuser = True
+admin.is_staff = True
+admin.first_name = 'Admin'
+admin.last_name = 'Adminsson'
+admin.set_password('admin')
+admin.save()
+Profile.objects.create(user=User.objects.get(username='admin'))
+users.append(admin)
+
+User.objects.create(username='mod', is_staff=True)
+users.append('mod')
+Profile.objects.create(user=User.objects.get(username='mod'))
+User.objects.create(username='csicska')
+users.append('csicska')
+Profile.objects.create(user=User.objects.get(username='csicska'))
+
+
+def generate_owner_votes():
+    owner = random.choice(users)
+    v = Votes()
+    v.save()
+    upvoters = random.sample(
+        users,
+        random.randint(0, len(users))
+    )
+    if owner in upvoters:
+        upvoters.remove(owner)
+    downvote_choices = [item for item in users if item not in upvoters]
+    downvoters = random.sample(
+        downvote_choices,
+        random.randint(0, len(downvote_choices))
+    )
+    for u in upvoters:
+        v.upvoters.add(Profile.objects.get(user__username=u))
+    for u in downvoters:
+        v.downvoters.add(Profile.objects.get(user__username=u))
+    return {
+        'owner': owner,
+        'votes': v
+    }
+
+
+# Accounts
+for i in range(random.choice(range(3, 9))):
+    # Users
+    fakedata = fake.simple_profile()
+    u = User()
+    u.username = fakedata['username']
+    name = fakedata['name'].split(' ')
+    u.first_name = name[0]
+    u.last_name = name[1]
+    u.save()
+    # Profiles
+    Profile.objects.create(user=u)
+    users.append(u.username)
+
+# Tags
+tag_choices = []
+for t in range(random.choice(range(5, 11))):
+    tag_choices.append(fake.word())
+
+# Questions
+for i in range(random.choice(range(3, 11))):
+    ov_q = generate_owner_votes()
+    q = Question()
```

Comment:
One letter variable names are ugly and indistinctive.

Reply:
Will be fixed later, the script is just a placeholder to test GET endpoints on question, answer and comment.